.github/workflows/lint.yaml

```diff
@@ -36,3 +31,17 @@ jobs:
             --target-branch ${{ github.event.repository.default_branch }} \
             --all \
             --validate-maintainers=false
+
+      - name: Run template validation
```

Comment:
Is it worth making this a reusable workflow that we could call from places that template capi-helm-charts values?

Reply:
On second thoughts, we could just add the same test to places that template capi-helm-charts values.yaml, like the operator and azimuth-ops